client/src/components/AppInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import { BodyShort, Label } from '@navikt/ds-react'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { HjelpemiddelItem } from '../interfaces/CommonTypes'
+
+type Props = {
+  hm: HjelpemiddelItem
+}
+
+export const AppInfo: React.FC<Props> = (props: Props) => {
+  const { t } = useTranslation()
+  const { hm } = props
+
+  return (
+    <>
+      <div style={{ marginBottom: '0.5rem' }}>
+        <Label style={{ display: 'inline' }}>{t('hjelpemiddelinfo.kalenderapp.utprøvingForBruker.label')} </Label>
+        <BodyShort style={{ display: 'inline' }}>
+          {hm.appInfo?.brukerHarProvdProvelisens
+            ? t('hjelpemiddelinfo.kalenderapp.utprøvingForBruker.harPrøvd')
+            : t('hjelpemiddelinfo.kalenderapp.utprøvingForBruker.harIkkePrøvd')}
+        </BodyShort>
+      </div>
+
+      <div style={{ marginBottom: '0.5rem' }}>
+        <Label style={{ display: 'inline' }}>{t('hjelpemiddelinfo.kalenderapp.støtteperson.label')} </Label>
+        <BodyShort style={{ display: 'inline' }}>
+          {hm.appInfo?.stottepersonSkalAdministrere
+            ? t('hjelpemiddelinfo.kalenderapp.støtteperson.skalAdministrere')
+            : t('hjelpemiddelinfo.kalenderapp.støtteperson.skalIkkeAdministrere')}
+        </BodyShort>
+      </div>
+
+      {hm.appInfo?.stottepersonSkalAdministrere && hm.appInfo.stottepersonHarProvdProvelisens != null && (
```

Comment:
Verdien fra APIet vil vel være `undefined` her. Så jeg vil heller foreslå å bruke typesjekk også, med 
```suggestion
      {hm.appInfo?.stottepersonSkalAdministrere && hm.appInfo.stottepersonHarProvdProvelisens !== undefined && (
```

Reply:
Var det eg også tenkte først, men når eg satte msw mocken til 
```
appInfo: {
    brukerHarProvdProvelisens: true,
    stottepersonSkalAdministrere: true,
    stottepersonHarProvdProvelisens: null
}
```
så blei teksten vist. Er det pga lureri med msw vs ekte api kall?

Reply:
Hm, men setter du `stottepersonHarProvdProvelisens: null` mockes vel ikke den faktiske responsen fra APIet? Den bør vel være `stottepersonHarProvdProvelisens: undefined`?

Reply:
Beklager, ser jo nå at APIet returnerer `null`. Da er det nok riktig sånn det var!